components/snackbar/snackbar.css

```diff
@@ -0,0 +1,54 @@
+.snackbar-content{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 40rem;
+    background-color: var(--background-black);
+    color: var(--secondary-text-color);
+    border-radius: 5px;
+    padding: 1rem;
+}
+
+.snackbar-button{
+    background-color: transparent;
+    color: var(--snackbar-text-color);
+    border: none;
+    font-size: 1.6rem;
```

Comment:
![image](https://user-images.githubusercontent.com/91944618/153699362-feb12376-beaf-4704-b7f3-5194771e9f38.png)

It's not readable if you could verify.

Reply:
Ok, I will check that, Thank you for the review